debian/postinst.d/utils/constants.sh

```diff
@@ -3,7 +3,7 @@
 # General configurations
 readonly SERVICE_NAME="ottopia-keepalived-exporter"
 readonly SERVICE_USER_NAME="ottopia_keepalived"
-readonly SERVICE_GROUP_NAMES="ottopia_keepalived"
+readonly SERVICE_GROUP_NAMES="ottopia_keepalived,docker"
```

Comment:
do we need this ? now that we do not use `docker`

Reply:
https://github.com/ottopia-tech/keepalived-exporter/pull/5/commits/3fe7b23a9bfaf673e7f88f58f39c7dae7827250f